### ISSUE TITLE
Unable to download archive even after it's available 

### ISSUE THREAD

**Issue by varunmalhotraminjar** [uskudnik/amazon-glacier-cmd-interface#178]
First I instructed Glacier to retrieve an archive using below command :
 
glacier-cmd getarchive <vault> <archive-ID>

After few hours I got an email notification about the completion of the retrival job. As the data is available for download I ran this command : 

glacier-cmd download <vault> <archive-ID>

All I'm seeing on the screen is some weird codes dancing on the STDOUT. Tried adding "--outputdirectory <dir_name> but it didn't work. Could someone please help!

**Comment by wvmarle** [comment]
It dumps the data to stdout. Use a redirect such as "> outfile" to tell the OS where to send the data to.

So the command would look about like this:
$ glacier-cmd download ... > archive.zip
with ... replaced by the other options such as the archive ID.

**Comment by varunmalhotraminjar** [comment]
Awesome!! Thank you so much. 
Does the outfile name "> archive.zip" has to be same as the file/archive real name/extension with which it was uploaded to vault?

**Comment by wvmarle** [comment]
No, any name will do. For your convenience, do use the correct extension, making it easier to recognise the file later.
In fact, when uploading a file to Glacier, the file name gets lost. Glacier stores the data and refers to it by the ArchiveID instead. That you can still see the file name in glacier-cmd is because it stores the file name and ArchiveID separately in a simpleDB database.

**Comment by varunmalhotraminjar** [comment]
Sure, Thank you! I'll try to install SimpleDB. 

One last question please, is there a way to get the Archive ID of the recently uploaded archive? Because, I just uploaded an archive of 4.2GB and all is shows is: 

[root@ip-10-12-111-2 data_back]# glacier-cmd upload howopensource db_bck.zip --description "Backup_Mon"

Wrote 4.2 GB. Rate 3.08 MB/s.. Rate 4.84 MB/s, average 3.08 MB/s, ETA 09:55:49.

Header (Archive SHA256 tree hash Uploaded File) and Value. 

Is it possible to get the archive ID?

**Comment by andrewgaul** [comment]
> Is it possible to get the archive ID?

#170 fixes the missing archive ID.  @uskudnik could you merge this?
